File: shivam-chaurasiya/Practice Problems/Level 1/01 Beginner Programs/Problem24.py
# BMI(Body Mass Index) Calculator
# Write a program that calculates Body Mass Index using: BMI = weight(kg) / (height(m))2


weight = float(input('Enter the weight(kg): '))
height = float(input('Enter the height(m): '))

bmi = weight / (height)**2
print(f'Body Mass Index is: {bmi:.2f}')

# Adding logic to display the category
if bmi < 18.5:
    print("Category: Underweight")
elif 18.5 <= bmi < 24.9:
    print("Category: Normal weight")
elif 25 <= bmi < 29.9:
    print("Category: Overweight")
else:
    print("Category: Obesity")

File: Basic Programs/Problem_24.js
/*--------------------------------
Problem 24: BMI Calculator

Write a program that calculates Body Mass Index using: BMI = weight(kg) / (height(m))²
--------------------------------*/

const weight = 68;
const height = 1.64;

const BMI = weight / height ** 2;

console.log(
  `Body Mass Index (BMI) for Weight ${weight}kg, Height ${height}m is: ${BMI}`,
);

/* OUTPUT:
Body Mass Index (BMI) for Weight 68kg, Height 1.64m is: 25.282569898869724
*/
